src/nav-bar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export default function Nav (props) {
+  return (
+    <nav className='navbar navbar-expand-lg navbar-light bg-warning navigation'>
+      <i className='fas fa-bolt text-dark nav-icon'></i>
+      <a className='navbar-brand text-dark float-left' href='#'>FlashApp</a>
+      <button className='navbar-toggler' type='button' data-toggle='collapse' data-target='#navbarNav' aria-controls='navbarNav' aria-expanded='false' aria-label='Toggle navigation'>
```

Comment:
Are you sure you need this button? It doesn't do anything. Don't just copy the examples from Bootstrap's docs.